src/redux/bids/index.ts

```diff
@@ -22,7 +20,7 @@ export interface BidsBySaleId {
   // "ox223123nlda": {"lastupdated", "bids: []"}
   [saleId: string]: {
     updatedAt: number // UTC timestamp
-    bids: SaleBid[] // bids
+    bids: GetAllBidsBySales_fixedPriceSale_commitments[] // bids
```

Comment:
I've kept it as bids since the redux component is supposed to work for all the upcoming sales. Plus the redux component will soon be deprecated anyway